tests/test_assets.py

```diff
@@ -172,6 +172,27 @@ def build_lookup_generic_cases(asset_finder_type):
 
 class AssetTestCase(TestCase):
 
+    asset_attrs = ['sid',
```

Comment:
Better yet, we should be dynamically generating this list from the types themselves:

```
In [1]: from zipline.assets import Asset
f
In [2]: from types import GetSetDescriptorType

In [3]: [name for name, value in vars(Asset).items() if isinstance(value, GetSetDescriptorType)]
Out[3]:
['auto_close_date',
 'sid',
 'start_date',
 'end_date',
 'exchange',
 'symbol',
 'first_traded',
 'asset_name']
```